wire-ios-data-model/Source/Model/FeatureConfig/AppLock/AppLockController.swift

```diff
@@ -20,8 +20,7 @@ import Foundation
 import LocalAuthentication
 
 public final class AppLockController: AppLockType {
-
-    static let log = ZMSLog(tag: "AppLockController")
+    let logger = WireLogger(tag: "AppLock")
```

Comment:
Why not adding a static property to `WireLogger` in `wire-ios-system/Source/Logging/WireLogger.swift` instead?